index.d.ts

```diff
@@ -0,0 +1,50 @@
+export as namespace EventEmitter;
+
+type ListenerFn = (...args: Array<any>) => void;
+
+/**
+ * Minimal `EventEmitter` interface that is molded against the Node.js
+ * `EventEmitter` interface.
+ */
+export class EventEmitter {
+  prefix: string;
```

Comment:
Good catch, also the correct name is `prefixed` and can be either a string or boolean (`false`).